### ISSUE TITLE
Error when installing magic-cli (IOError)

### ISSUE THREAD

**Issue by AshtonHamm** [magic-network/magic-cli#10]
Operating System : Kubuntu 18.04
No virtual environment being used for this project at the moment.

Tried to install the magic-cli exactly as described in the documentation. However, ran into the error below when executing the "pip install . " inside of the magic-cli directory:

    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-5q7f9i/pyobjc-framework-CoreData/setup.py", line 27, in <module>
        if fn.startswith('_CoreData')
      File "pyobjc_setup.py", line 389, in Extension
        os_level = get_os_level()
      File "pyobjc_setup.py", line 203, in get_os_level
        pl = plistlib.readPlist('/System/Library/CoreServices/SystemVersion.plist')
      File "/usr/lib/python2.7/plistlib.py", line 75, in readPlist
        pathOrFile = open(pathOrFile)
    IOError: [Errno 2] No such file or directory: '/System/Library/CoreServices/SystemVersion.plist'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-5q7f9i/pyobjc-framework-CoreData/

**Comment by AshtonHamm** [comment]
Tried installing a virtual environment to handle any issues and upgraded my setup tools with the following commands:

`python3 -m venv Magic-Virt-Env`
`source Magic-Virt-Env/bin/activate`

Although, it still gives me the same error as before.
I just realized it could be because I am using Linux, and not OSX as mentioned above.
The error originates from the lack of an iOS file, suggesting OSX only (Source and text posted below):

**/System/Library/CoreServices/SystemVersion.plist**

_This file is where iOS gets the firmware version, build number, and build type. Editing this file is often done to fake the firmware version (to trick anything that has a hardcoded check on it), changing the build type from "Beta" to something else on beta builds (to prevent expiration), and more._ 

Source URL: https://www.theiphonewiki.com/wiki//System/Library/CoreServices/SystemVersion.plist

So, my bad! That is what I get for trying to use an OSX application with a Linux system without realizing it only says "macOS 10.6 or higher".

Sorry for any unnecessary confusion, but hope this helps others who might have done this by mistake.
